src/frontend/pages/result/result.js

```diff
@@ -17,13 +17,13 @@ export function Result(){
             
             <div className="result-page">
                 <strong className="result-title">Quiz Result</strong>
-                <section className="score-section">Congratulations you scored : {score<1 ? "0"+score : score*10} out of {categoryQuestions.length*10}</section>
+                <section className="score-section">{ score*10> categoryQuestions.length*10/2 ? "Congratulations 🎉, " : "Give it try once 😔, " }you scored {score<1 ? "0"+score : score*10} out of {categoryQuestions.length*10}</section>
 
                 <div className="question-container flex-col">
                     {categoryQuestions.map((item)=>{const {questionText, options, userAnswer,_id}=item;
```

Comment:
You can destructure the `item` in the arguments itself. No need to do it like you have done here.
`____.map( //destructure here itself=>(<div></div>)`

Reply:
done refactored destructure part